packages/graphql-parse-resolve-info/README.md

```diff
@@ -372,6 +372,26 @@ single-level object containing only the fields compatible with
            ...as before...
 ```
 
+### `isResolveTree(value)`
+
+Determines whether the argument is a value of type `ResolveTree`. This is useful only to TypeScript users. It allows to differentiate between `ResolveTree` and `FieldsByTypeName` objects to pass to `simplifyParsedResolveInfoFragmentWithType `.
+
+Example:
+
+```ts
+import { isResolveTree } from "graphql-parse-resolve-info";
+
+const { parsedResolveInfoFragment, simplifiedFragment } = await graphql(
+  Schema,
+  query
+);
```

Comment:
I think you might have mis-typed this example, we wouldn't get these results from calling `graphql()`?

Maybe something like this would be more helpful?

```suggestion
const myResolver: GraphQLFieldResolver<any, any> = (source, args, context, resolveInfo) => {
  const parsedResolveInfoFragment = parseResolveInfo(resolveInfo);
  if (!isResolveTree(parsedResolveInfoFragment)) {
    throw new Error(`Expected ResolveTree`);
  }
  const simplifiedFragment = simplifyParsedResolveInfoFragmentWithType(
    parsedResolveInfoFragment,
    resolveInfo.returnType
  );
  // ...
};
```